fixtures/american-dj/vizi-q-wash7.json

```
@@ -0,0 +1,851 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Vizi Q Wash7",
+  "shortName": "Q7",
+  "categories": ["Moving Head", "Color Changer"],
+  "meta": {
+    "authors": ["Rey Victor", "Luke Nelson"],
+    "createDate": "2022-12-28",
+    "lastModifyDate": "2022-12-28"
+  },
+  "comment": "The Vizi Q Wash7 is a 280W professional moving head wash fixture with variable zoom (5 ~ 55-degrees) and seven 40W Quad (RGBW) LEDs. At 280W, this fixture offers very powerful 5-degree beams or brilliant 55-degree washes of color. WiFLY wireless DMX on-board.",
+  "links": {
+    "manual": [
+      "https://d295jznhem2tn9.cloudfront.net/ItemRelatedFiles/10532/ADJ%20VIZI%20Q%20WASH7%20-%20USER%20MANUAL%20.pdf"
+    ],
+    "productPage": [
+      "https://www.adj.com/vizi-q-wash7"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=EHeTZ_G_8zM"
+    ]
+  },
+  "physical": {
+    "dimensions": [284.4, 411.7, 202],
+    "weight": 8.5,
+    "power": 280,
+    "DMXconnector": "3-pin and 5-pin",
+    "bulb": {
+      "type": "LED"
+    },
+    "lens": {
+      "degreesMinMax": [7, 58]
+    }
+  },
+  "matrix": {
+    "pixelKeys": [
+      [
+        [null, "O1", null, "O2", null],
+        ["O3", null, "C", null, "O4"],
+        [null, "O5", null, "O6", null]
+      ]
+    ],
+    "pixelGroups": {
+      "All": "all",
+      "Center": { "name": ["C"] },
+      "Outer": { "name": ["^O\\d+$"] }
+    }
+  },
+  "templateChannels": {
+    "Red $pixelKey": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green $pixelKey": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue $pixelKey": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "White $pixelKey": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    }
+  },
+  "availableChannels": {
+    "Pan": {
+      "fineChannelAliases": ["Pan fine"],
+      "defaultValue": "50%",
+      "capability": {
+        "type": "Pan",
+        "angleStart": "0deg",
+        "angleEnd": "630deg"
+      }
+    },
+    "Tilt": {
+      "fineChannelAliases": ["Tilt fine"],
+      "defaultValue": "50%",
+      "capability": {
+        "type": "Tilt",
+        "angleStart": "-160deg",
+        "angleEnd": "160deg"
+      }
+    },
+    "Color Macros": {
+      "defaultValue": "0%",
+      "capabilities": [
+        {
+          "dmxRange": [0, 0],
+          "type": "ColorPreset",
+          "colors": ["#000000"],
+          "comment": "OFF"
+        },
```

This is probably a `NoFunction` capability; otherwise it would override the RGB channels.

```suggestion
        {
          "dmxRange": [0, 0],
          "type": "NoFunction"
        },
```